app/assets/javascripts/application.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+document.onreadystatechange = function () {
+  if (document.readyState === 'interactive') {
+    var editDocumentForm = document.querySelector('.edit_document')
+    if (editDocumentForm == null) { return } // only run on edit document page
+    var urlPreview = document.getElementById('url-preview-id')
+    var basePath = document.getElementById('base-path-id')
+    var documentTitle = document.getElementById('document-title-id')
+    var noTitle = document.getElementById('no-title-id')
+    var errorGeneratingPath = document.getElementById('error-generating-path-id')
+    var showErrorMessage = function () {
+      urlPreview.setAttribute('class', 'app-hidden')
+      noTitle.setAttribute('class', 'app-hidden')
+      errorGeneratingPath.removeAttribute('class')
+    }
+    documentTitle.onblur = function () {
+      var url = new URL(document.location.origin + editDocumentForm.getAttribute('data-generate-path-path'))
+      if (!documentTitle.value) {
+        noTitle.removeAttribute('class')
```

Comment:
Could we replace this code with a call to 'showErrorMessage'?

Reply:
This shows a different message for the case where a user deletes the title in the box. It's not the same as the error message.